momentum/changelog/4/4-3-0.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Momentum 4.3.0 Changelog"
-description: "Momentum 4.3.0 was released on 2020-01-07. This section will list all of the major changes that happened with the release of Momentum 4.3.0 Depending on installation type, all changes may not be applicable"
+description: "Momentum 4.3.0 was released on 2019-01-07. This section will list all of the major changes that happened with the release of Momentum 4.3.0 Depending on installation type, all changes may not be applicable"
 ---
 
 **Momentum 4.3.0** was released on 2019-3-31. This section will list all of the major changes that happened with the release of **Momentum 4.3.0**. Depending on installation type, all changes may not be applicable
```

Comment:
This date, and the one above are still not agreeing.  I'm pretty sure the 2019-3-31 date is the correct one.